nanoFramework.Logging.Syslog/SyslogClient.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
```

Comment:
And here :-) I guess most likely in other places. Please add them in all the cs file